cluster/examples/kubernetes/ceph/common.yaml

```diff
@@ -1631,6 +1631,9 @@ rules:
   - apiGroups: ["storage.k8s.io"]
     resources: ["volumeattachments"]
     verbs: ["get", "list", "watch", "update", "patch"]
+  - apiGroups: ["storage.k8s.io"]
```

Comment:
@agarwal-mudit forgot to mention, same changes are required in https://github.com/rook/rook/blob/master/tests/framework/installer/ceph_manifests.go

Reply:
fixed, thanks.